packages/agents-openai/test/openaiResponsesModel.helpers.test.ts

```diff
@@ -203,7 +203,17 @@ describe('getInputItems', () => {
         providerData: { type: 'web_search' },
       },
     ] as any);
-    expect(web[0]).toMatchObject({ type: 'web_search' });
+    expect(web[0]).toMatchObject({ type: 'web_search_call' });
+
+    const webCall = getInputItems([
+      {
+        type: 'hosted_tool_call',
+        id: 'w',
+        status: 'completed',
+        providerData: { type: 'web_search_call' },
+      },
+    ] as any);
+    expect(webCall[0]).toMatchObject({ type: 'web_search_call' });
```

Comment:
the type here must be consistently *_call for OAI Responses API compatibility, so this behavior changes are appropriate.